InfiniteLoopBookProject/tests/TestDAO.java

```diff
@@ -0,0 +1,4 @@
+

```

Comment:
This and `TestService` should either be implemented or removed. If left in, they should probably be made `final`.

Reply:
Same with [Test Service](https://github.com/STOzaki/InfiniteLoop/blob/b4b4c36303b919901d9f3902947f37dfbf5bab0b/InfiniteLoopBookProject/tests/TestService.java#L2)

Reply:
Everything is well done. I approve.